profiles/base/package.use.mask

```diff
@@ -10,6 +10,10 @@
 # app-text/gocr has been masked for removal, bug #824290
 media-gfx/xsane ocr
 
+# Takuya Wakazono <pastalian46@gmail.com> (2023-11-21)
```

Comment:
Given rpm-sequoia needs Rust, let's do this in the wd40 profiles (as well).

Reply:
done